packages/embeds/embed-core/src/embed.ts

```diff
@@ -440,16 +440,20 @@ export class Cal {
     return Object.fromEntries(Object.entries(params).filter(([key, value]) => !excludeParam(key, value)));
   }
 
+  private getQueryParamsFromPage() {
+    const queryParamsFromPage = getQueryParamsFromPage();
+    // Ensure valid params are used from the page.
+    return this.filterParams(queryParamsFromPage);
```

Comment:
Filtering just the page params now. So, config goes unfiltered as that is explicitly passed by user.

Page Params could be unintended and thus need to be filtered.